src/screens/Ticketing/Purchase/Overview/index.tsx

```diff
@@ -83,6 +83,27 @@ const PurchaseOverview: React.FC<OverviewProps> = ({
     defaultUserProfilesWithCount,
   );
 
+  const [selectableUserProfiles, setSelectableUserProfiles] = useState(
+    defaultUserProfilesWithCount,
+  );
+
+  useEffect(() => {
+    const options = defaultUserProfilesWithCount.filter((p) => {
```

Comment:
Blir dette riktig? Hva hvis 7-dagers hadde vært default produkt, da hadde defaultUserProfiles blitt uten "Student", og da aldri blitt med i filtreringen her? 🤔

Reply:
Mulig jeg misforstår, men sånn som det funker nå, så kan jeg sette 7-dagers som default valg på produkt. Har satt default profile til å være student. Initelt så vil viewet vise en 7 dagers billett for voksen, men hvis en bytte til en billettype som støtter student, så blir det mulig å velge student igjen 😊 Så jeg tror det funker som forventet?